From c09022a66e08bb96cf7b40830e7eec639bebf7a0 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Mon, 21 Jul 2014 10:34:11 +0800
Subject: [PATCH 2/2] boot/dts: increase uboot size for fsl-e500mc BSP

Increase the uboot size from 512k to 768k for BSP fsl-e500mc dts files
p2041rdb.dts & p3041ds.dts & p4080ds.dts.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/powerpc/boot/dts/p2041rdb.dts |   22 +++++++++++-----------
 arch/powerpc/boot/dts/p3041ds.dts  |   22 +++++++++++-----------
 arch/powerpc/boot/dts/p4080ds.dts  |   22 +++++++++++-----------
 3 files changed, 33 insertions(+), 33 deletions(-)

diff --git a/arch/powerpc/boot/dts/p2041rdb.dts b/arch/powerpc/boot/dts/p2041rdb.dts
index 32eba4c..651f8a6 100644
--- a/arch/powerpc/boot/dts/p2041rdb.dts
+++ b/arch/powerpc/boot/dts/p2041rdb.dts
@@ -306,31 +306,31 @@
 			};
 			partition@3f80000 {
 				label = "alt-u-boot";
-				reg = <0x3f80000 0x80000>;
+				reg = <0x3f80000 0xc0000>;
 				read-only;
 			};
-			partition@4000000 {
+			partition@4020000 {
 				label = "alt-rcw";
-				reg = <0x4000000 0x20000>;
+				reg = <0x4020000 0x20000>;
 				read-only;
 			};
-			partition@4020000 {
+			partition@4040000 {
 				label = "fs2";
-				reg = <0x4020000  0x3f20000>;
+				reg = <0x4040000  0x3ea0000>;
 			};
-			partition@7f40000 {
+			partition@7f00000 {
 				label = "fman-firmware";
-				reg = <0x7f40000 0x20000>;
+				reg = <0x7f00000 0x20000>;
 				read-only;
 			};
-			partition@7f60000 {
+			partition@7f20000 {
 				label = "u-boot env";
-				reg = <0x7f60000  0x20000>;
+				reg = <0x7f20000  0x20000>;
 				read-only;
 			};
-			partition@7f80000 {
+			partition@7f40000 {
 				label = "u-boot";
-				reg = <0x7f80000 0x80000>;
+				reg = <0x7f40000 0xc0000>;
 				read-only;
 			};
 		};
diff --git a/arch/powerpc/boot/dts/p3041ds.dts b/arch/powerpc/boot/dts/p3041ds.dts
index 5ed6b48..a299353e 100644
--- a/arch/powerpc/boot/dts/p3041ds.dts
+++ b/arch/powerpc/boot/dts/p3041ds.dts
@@ -293,31 +293,31 @@
 			};
 			partition@3f80000 {
 				label = "alt-u-boot";
-				reg = <0x3f80000 0x80000>;
+				reg = <0x3f80000 0xc0000>;
 				read-only;
 			};
-			partition@4000000 {
+			partition@4020000 {
 				label = "alt-rcw";
-				reg = <0x4000000 0x20000>;
+				reg = <0x4020000 0x20000>;
 				read-only;
 			};
-			partition@4020000 {
+			partition@4040000 {
 				label = "fs2";
-				reg = <0x4020000  0x3f20000>;
+				reg = <0x4040000  0x3ea0000>;
 			};
-			partition@7f40000 {
+			partition@7f00000 {
 				label = "fman-firmware";
-				reg = <0x7f40000 0x20000>;
+				reg = <0x7f00000 0x20000>;
 				read-only;
 			};
-			partition@7f60000 {
+			partition@7f20000 {
 				label = "u-boot env";
-				reg = <0x7f60000  0x20000>;
+				reg = <0x7f20000  0x20000>;
 				read-only;
 			};
-			partition@7f80000 {
+			partition@7f40000 {
 				label = "u-boot";
-				reg = <0x7f80000 0x80000>;
+				reg = <0x7f40000 0xc0000>;
 				read-only;
 			};
 		};
diff --git a/arch/powerpc/boot/dts/p4080ds.dts b/arch/powerpc/boot/dts/p4080ds.dts
index 26ada2a..cc6dc2e 100644
--- a/arch/powerpc/boot/dts/p4080ds.dts
+++ b/arch/powerpc/boot/dts/p4080ds.dts
@@ -316,31 +316,31 @@
 			};
 			partition@3f80000 {
 				label = "alt-u-boot";
-				reg = <0x3f80000 0x80000>;
+				reg = <0x3f80000 0xc0000>;
 				read-only;
 			};
-			partition@4000000 {
+			partition@4020000 {
 				label = "alt-rcw";
-				reg = <0x4000000 0x20000>;
+				reg = <0x4020000 0x20000>;
 				read-only;
 			};
-			partition@4020000 {
+			partition@4040000 {
 				label = "fs2";
-				reg = <0x4020000  0x3f20000>;
+				reg = <0x4040000  0x3ea0000>;
 			};
-			partition@7f40000 {
+			partition@7f00000 {
 				label = "fman-firmware";
-				reg = <0x7f40000 0x20000>;
+				reg = <0x7f00000 0x20000>;
 				read-only;
 			};
-			partition@7f60000 {
+			partition@7f20000 {
 				label = "u-boot env";
-				reg = <0x7f60000  0x20000>;
+				reg = <0x7f20000  0x20000>;
 				read-only;
 			};
-			partition@7f80000 {
+			partition@7f40000 {
 				label = "u-boot";
-				reg = <0x7f80000 0x80000>;
+				reg = <0x7f40000 0xc0000>;
 				read-only;
 			};
 		};
-- 
1.7.5.4

